src/Bicep.Core/Extensions/JsonElementExtensions.cs

```diff
@@ -25,6 +26,7 @@ public static class JsonElementExtensions
         public static string ToNonNullString(this JsonElement element) =>
             element.GetString() ?? throw new JsonException($"Expected \"{element}\" to be non-null.");
 
+        [SuppressMessage("Trimming", "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code", Justification = "<Pending>")]
```

Comment:
Are you planning to either add the `<Pending>` justifications or addressing before merging?

Reply:
Yeah, good catch.

Reply:
Done